catroid/src/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -66,6 +68,13 @@ protected void onCreate(Bundle savedInstanceState) {
 		webView.setWebViewClient(new MyWebViewClient());
 		webView.getSettings().setJavaScriptEnabled(true);
 
+		String language = UtilDeviceInfo.getUserLanguageCode(this);
```

Comment:
The language should be the `Catrobat language version`, not the locale of the device. Please update your code.